core-tests/src/test/java/org/mule/runtime/core/internal/transformer/simple/LocalDateToDateTestCase.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.core.internal.transformer.simple;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+import org.mule.tck.junit4.AbstractMuleContextTestCase;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.Date;
+
+import org.junit.Test;
+
+public class LocalDateToDateTestCase extends AbstractMuleContextTestCase {
```

Comment:
No value in extending AbstractTransformerTestCase, like other transformer tests?